packages/gonx/src/generators/go-blueprint/go-blueprint.ts

```diff
@@ -60,20 +63,22 @@ function executeGoBlueprintCommand(
 
       const childProcess = fork(goBlueprintBin, args, {
         cwd: executionDir,
-        stdio: ['pipe', 'ignore', 'pipe', 'ipc'],
+        stdio: isWindows ? 'inherit' : ['pipe', 'ignore', 'pipe', 'ipc'],
```

Comment:
suggestion: as discussed, let's simplify this by replacing the ignore with a pipe and see if it works.